Source/WebKit/WebProcess/cocoa/PlaybackSessionManager.messages.in

```diff
@@ -39,6 +39,8 @@ messages -> PlaybackSessionManager {
     SelectLegibleMediaOption(WebKit::PlaybackSessionContextIdentifier contextId, uint64_t index)
     HandleControlledElementIDRequest(WebKit::PlaybackSessionContextIdentifier contextId)
     TogglePictureInPicture(WebKit::PlaybackSessionContextIdentifier contextId)
+    ToggleInWindow(WebKit::PlaybackSessionContextIdentifier contextId)
+    IsInWindowActive(WebKit::PlaybackSessionContextIdentifier contextId)
```

Comment:
This should change to a push message from the WebContent process to the UIProcess via the PlaybackSessionManagerProxy. When the WebContent process detects a change it in-window mode, it should send a message up.